src/python/pants/backend/python/goals/pytest_runner.py

```diff
@@ -133,6 +141,15 @@ async def setup_pytest_for_target(
         ),
     )
 
+    config_digest_request = Get(
+        Digest,
+        PathGlobs(
+            globs=[pytest.config] if pytest.config else [],
+            glob_match_error_behavior=GlobMatchErrorBehavior.error,
+            description_of_origin="the option `--pytest-config`",
```

Comment:
People are far more likely to express this in Pants config, so I'd say maybe `[pytest].config`?

Reply:
Agreed, although I'm working on a PR to fix this across the board + improve the error message for unmatched glos. Currently, all the other subsystems use `--pytest-config` syntax so it's fine for now, and then I'll fix.